src/components/home/SearchBottom.tsx

```diff
@@ -0,0 +1,201 @@
+import Button from '../common/Button';
+import { CloseIcon, MinusFilledIcon, MinusUnfilledIcon, PlusFilledIcon } from '@/assets/svg';
+import { flexCssGenerator } from '@/styles/customStyle.ts';
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+
+type SearchBottomProps = {
+	numParentsAdults: number;
+	setParentsNumAdults: React.Dispatch<React.SetStateAction<number>>;
+	onClose: () => void;
+};
+
+const SearchBottom: React.FC<SearchBottomProps> = ({ numParentsAdults, setParentsNumAdults, onClose }) => {
```

Comment:
p2: `React.FC`를 사용하지 않고, 다음과 같이 사용할 수 있습니다!

```suggestion
const SearchBottom = ({ numParentsAdults, setParentsNumAdults, onClose }: SearchBottomProps) => {
```